src/init/ext_manifest.c

```diff
@@ -85,10 +85,14 @@ const struct ext_man_dbg_abi ext_man_dbg_info
 	},
 };
 
+/* increment this value after adding any element to ext_man_config dictionary */
+#define CONFIG_ELEM_CNT 2
+
 const struct ext_man_config_data ext_man_config
 	__aligned(EXT_MAN_ALIGN) __section(".fw_metadata") = {
 	.hdr.type = EXT_MAN_ELEM_CONFIG_DATA,
-	.hdr.elem_size = ALIGN_UP(sizeof(struct ext_man_config_data),
+	.hdr.elem_size = ALIGN_UP(sizeof(struct ext_man_config_data) +
+				  sizeof(struct config_elem) * CONFIG_ELEM_CNT,
```

Comment:
Just wondering here if we can use a linker variable here for CONFIG_ELEM_COUNT as a way to automate this as it may be error prone for developers i.e. something cmake can pass into the linker pre-processor and have the linker set this based on the FW binary build ???

Reply:
@ktrzcinx agreed with Liam, we can try to use some trick with `enum` like 
```suggestion
				  sizeof(struct config_elem) * (EXT_MAN_CONFIG_LAST_ELEM - 1),
```

You do not need to update the `EXT_MAN_CONFIG_LAST_ELEM` or `CONFIG_ELEM_CNT` if we add a new type